packages/fondue/src/components/RichTextEditor/Plugins/BlurOnBreakPlugin/createBlurOnBreakPlugin.ts

```diff
@@ -6,9 +6,10 @@ import { BlurOnBreakPlatePlugin } from './types';
 
 export const KEY_BLUR_ON_BREAK = 'blurOnBreak';
 
-export const createBlurOnBreakPlugin = createPluginFactory<BlurOnBreakPlatePlugin>({
-    key: KEY_BLUR_ON_BREAK,
-    handlers: {
-        onKeyDown: onKeyDownBlurOnBreak,
-    },
-});
+export const createBlurOnBreakPlugin: ReturnType<typeof createPluginFactory> =
```

Comment:
```suggestion
export const createBlurOnBreakPlugin: ReturnType<typeof createPluginFactory<createPluginFactory>> =
```
Not sure if required but this would fix the typecasting in the index file of this plugin

Reply:
Added typing to avoid casting